packages/ui-heading/src/Heading/__examples__/Heading.examples.ts

```diff
@@ -32,26 +34,19 @@ export default {
       'inherit'
     ]
   },
-  // @ts-expect-error ts-migrate(6133) FIXME: 'props' is declared but its value is never read.
-  getComponentProps: (props) => {
+  getComponentProps: () => {
     return {
       children: 'Hello this is a test heading',
       margin: 'medium none'
     }
   },
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'props' implicitly has an 'any' type.
-  getExampleProps: (props) => {
+  getExampleProps: (props: HeadingProps) => {
     return {
-      background: ['primary-inverse', 'secondary-inverse'].includes(props.color)
+      background: ['primary-inverse', 'secondary-inverse'].includes(
+        props.color!
+      )
         ? 'primary-inverse'
         : 'primary'
     }
-  },
-  // @ts-expect-error ts-migrate(7006) FIXME: Parameter 'props' implicitly has an 'any' type.
-  filter: (props) => {
-    return (
-      (props.type === 'notification' && props.countUntil) ||
-      (props.standalone && props.placement !== 'start top')
-    )
```

Comment:
Lol these are the props of Badge, seems like this has been like this forever, since the component was added :D Good thing that TS catches these :)